frontend/apps/customer/.env.example

```
@@ -5,6 +5,14 @@ ENABLE_CONSOLE_LOGGING=true
 SKIP_ENV_VALIDATION=false
 HOTJAR_ENABLED=false
 MATOMO_ENABLED=false
-SENTRY_DSN=
+SENTRY_ORG=city-of-helsinki
+SENTRY_PROJECT=tilavarauspalvelu-customer-ui
+NEXT_PUBLIC_SENTRY_PROJECT=${SENTRY_PROJECT}
 SENTRY_AUTH_TOKEN=
-SENTRY_ENVIRONMENT=local
+SENTRY_ENABLE_SOURCE_MAPS=false
+NEXT_PUBLIC_SENTRY_TRACE_PROPAGATION_TARGETS=https://localhost:8000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_TRACE_PROPAGATION_TARGETS key should go before the SENTRY_AUTH_TOKEN key

(UnorderedKey)